src/main/frontend/public/index.html

```diff
@@ -8,6 +8,7 @@
     <!-- <meta firstName="viewport" content="user-scalable=0;"/> -->
     <meta name="theme-color" content="#000000" />
     <meta name="AIC Group" content="AIC Group" />
+    <meta name="description" content="Pioneers in dedicated service to the scientific community’s necessities by being a one stop Shoppe for all laboratory requisites." />

```

Comment:
Can you update the desc to "Associated Instruments and Chemicals popularly known as AIC, is a five decade old company. Pioneers in serving the scientific community by being a one stop solution for all the scientific needs."

Reply:
Associated Instruments and Chemicals popularly known as AIC, is a pioneer in serving the scientific community by being a one stop solution for all the scientific needs.